py/yubikey.py

```diff
@@ -327,6 +327,42 @@ def refresh_devices(self, reader_filter=None):
                         break
             return success({'devices': self._devices})
 
+    def write_config(self, usb_applications, nfc_applications):
+
+        usb_enabled = 0x00
+        nfc_enabled = 0x00
+        for app in usb_applications:
+            usb_enabled |= APPLICATION[app]
+        for app in nfc_applications:
+            nfc_enabled |= APPLICATION[app]
+
+        with open_device(serial=self._current_serial) as dev:
+
+            try:
+                dev.write_config(
+                    device_config(
+                        usb_enabled=usb_enabled,
+                        nfc_enabled=nfc_enabled,
+                        ),
+                    reboot=True)
+            except APDUError as e:
```

Comment:
If we are doing a version without supporting configuration lock, these specific error handling block should not be needed.